docs/alerts.md

```diff
@@ -0,0 +1,102 @@
+# Alerts
+
+<!-- TOC depthTo:2 -->
+
+- Elasticsearch cluster
+    - [Elasticsearch Cluster Health is Red](#Elasticsearch-Cluster-Health-is-Red)
+    - [Elasticsearch Cluster Healthy is Yellow](#Elasticsearch-Cluster-Healthy-is-Yellow)
+    - [Elasticsearch Write Requests Rejection Jumps](#Elasticsearch-Write-Requests-Rejection-Jumps)
+    - [Elasticsearch Node Disk Low Watermark Reached](#Elasticsearch-Node-Disk-Low-Watermark-Reached)
+    - [Elasticsearch Node Disk High Watermark Reached](#Elasticsearch-Node-Disk-High-Watermark-Reached)
+    - [Elasticsearch Node Disk Flood Watermark Reached](#Elasticsearch-Node-Disk-Flood-Watermark-Reached)
+    - [Elasticsearch JVM Heap Use is High](#Elasticsearch-JVM-Heap-Use-is-High)
+    - [Aggregated Logging System CPU is High](#Aggregated-Logging-System-CPU-is-High)
+    - [Elasticsearch Process CPU is High](#Elasticsearch-Process-CPU-is-High)
+    - [Elasticsearch Disk Space is Running Low](#Elasticsearch-Disk-Space-is-Running-Low)
+    - [Elasticsearch FileDescriptor Usage is high](#Elasticsearch-FileDescriptor-Usage-is-high)
+
+<!-- /TOC -->
+
+## Elasticsearch Cluster Health is Red
+
+At least one primary shard and its replicas are not allocated to a node.
+
+### Troubleshooting
+
+Cluster nodes could be failed or the Elasticsearch process crashes due to heavy load.
+
+## Elasticsearch Cluster Healthy is Yellow
+
+Replica shards for at least one primary shard aren't allocated to nodes.
```

Comment:
s/aren't/isn't/

Reply:
Fixed

Reply:
i clearly didn't read the whole sentence before i made this comment... you were correct before with "aren't"

Reply:
Np. Updated